test/services/DateServiceSpec.scala

```diff
@@ -50,25 +51,10 @@ class DateServiceSpec extends TestSupport with FeatureSwitching {
     }
   }
 
-  "The getCurrentTaxYearEnd method when TimeMachineAddYear FS is on" should {
-    "return the next year if the current date is before april 6 and the timeMachineAddYears 1" in {
-      enable(TimeMachineAddYear)
-      TestDateService.getCurrentTaxYearEnd(LocalDate.parse("2018-03-29")) shouldBe 2019
+  "The getCurrentTaxYearEnd" should {
```

Comment:
* I would need to rewire these unit test to use ~.getCurrentDate method;
* the best way test I would think to test getCurrentDate would be that if LocalDate.getYear return currentYear + 1 when timeMachine is on;